plugins/repository-s3/build.gradle

```diff
@@ -23,7 +23,7 @@ esplugin {
 }
 
 versions << [
-  'aws': '1.10.69'
+  'aws': '1.11.187'
```

Comment:
AWS SDK v2.0 is in beta, but is there a reason that we don't update to the latest version (1.11.211, maybe 1.11.212)?

Reply:
We wanted for this PR to use the same version that we have been using for ec2. Not mandatory though that we keep that consistent.
I'm all +1 to move to latest and also move S3 to latest version.